compress/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "include-flate-compress"
+version = "0.1.0"
+authors = ["SOFe <sofe2038@gmail.com>", "Kento Oki <hrn832@protonmail.com>"]
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/SOF3/include-flate.git"
+homepage = "https://github.com/SOF3/include-flate"
+description = "Compression algorithm provider"
+
+[dependencies]
+libflate = "2.0.0"
```

Comment:
I think we still benefit from cfg features to avoid compiling unused algorithms though.

Reply:
So you mean the user could *optionally* disable specific dependency for algorithms by their manual choise, if they don't use it, right?
Then making some features in both compression library crate and root crate like this would be an ideal option as previously implemented in early commits in this PR:

```toml
libflate = { version = "2.0.0", optional = true }
zstd = { version = "0.13.0", optional = true }

[features]
default = ["deflate", "zstd"]
deflate = ["dep:libflate"]
zstd = ["dep:zstd"]

# include-flate-compress/Cargo.toml
```

```toml
[features]
default = ["deflate", "zstd"]
deflate = ["include-flate-compress/deflate"]
zstd = ["include-flate-compress/zstd"]

# include-flate/Cargo.toml
```

And then, They're both set enabled by default, if user doesn't care about `deflate` method, like this:

```toml
[dependencies]
include-flate = { version = "...", default-features = false, features = ["zstd"] }
```

And vice versa:

```toml
[dependencies]
include-flate = { version = "...", default-features = false, features = ["deflate"] }
```

Reply:
Done.